src/main/resources/db/changelog/33-remove-years-column-from-experience.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE experience
+DROP COLUMN years;
```

Comment:
This way we are going to lose the data for all years. Even tho this is a test database it will make all records with similar experience (up to 11 months) and we won't be able to test properly.
Suggestion - try to add this before dropping the years and see if it will do the job: 
```
UPDATE experience
SET months = (years * 12) + months;
```